mne/source_space.py

```diff
@@ -2306,24 +2327,105 @@ def get_volume_labels_from_aseg(mgz_fname):
 
     Returns
     -------
-    label_names : list of str
+    label_names_sorted : list of str
         The names of segmented volumes included in this mgz file.
-
+    label_colors_sorted : list of str
+        The RGB colors of the labels included in this mgz file.
```

Comment:
add a return_colors=True | False parameter otherwise you break the API and backward compat of MNE